docs/cli/new.md

```diff
@@ -39,25 +44,53 @@ pulls a nio project from the `bar` repository of a GitHub user `foo` and names i
 
 ---
 
+## Example (configure Pubkeeper):
+
+```bash
+nio new my_project --pubkeeper-hostname <system_id>.pubkeeper.nio.works --pubkeeper-token <pubkeeper_token>
+```
+
+creates a new folder in your current directory named `my_project` with pubkeeper settings configured for system wide communication.
+
+---
+
+## Example (configure authentication):
+
+```bash
+nio new my_project --username <username> --password <password>
+```
+
+creates a new folder in your current directory named `my_project` with a specified user created to access this instance. Use your configured `<username>` and `<password>` values in the [System Designer](/system-designer/reference.md#connect-to-a-local-instance).
+
+---
+
+## Example (configure encryption):
+
+```bash
+nio new my_project --ssl
+```
+
+creates a new folder in your current directory named `my_project`. Follow the prompts to configure your instance encryption
+
+---
+
 ## New project creation
 
-Once you've executed one of the comands above, follow the prompts to configure your project. You can always open of nio.conf later to modify any values, but the nio-cli takes care of it for you automatically.
+Once you've executed one of the commands above, follow the prompts to configure your project. You can always open of nio.conf later to modify any values, but the nio-cli takes care of it for you automatically.
 
 >**[info] default values**
 >
 >* if a prompt defines a default value, you may hit `Enter` to accept it.
 
 **Configure your local nio instance:**
-- Enter instance hostname or IP \[default: localhost\]:
-- Enter instance port \[number, below 1024 requires "sudo", default: 8181\]:
+- `nio new <project> --ip <ip> --port <port>`
+- Instance hostname or IP \[default: localhost\]:
+- Instance port \[number, below 1024 requires "sudo", default: 8181\]:
```

Comment:
```
C:\Users\Tom>sudo
'sudo' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
Would `requires superuser access` be better here?